1518.py

```diff
@@ -0,0 +1,16 @@
+class Solution:
+    def numWaterBottles(self, numBottles: int, numExchange: int) -> int:
+        total, left_empty, to_drink = 0, 0, numBottles
```

Comment:
Function parameters provided by LeetCode don't follow Python naming conventions. We should rename `numBottles` and `numExchange` such that they are consistent with the other variables in the solution.